Crawl/CognitiveServiceEntityLinking/CognitiveServiceEntityLinking.cs

```diff
@@ -39,27 +39,35 @@ public static Task<HttpResponseMessage> Run(HttpRequestMessage req, TraceWriter
                     if (textBuilder.Length == 0)
                         return null;
 
-                    return Services.Limit(textBuilder.ToString(), 10240);
+                    string text = Services.Limit(textBuilder.ToString(), 10240);
```

Comment:
>10240 [](start = 73, length = 5)

Does it make sense to introduce some constants here with additional documentation?  A link to spec perhaps?

Reply:
Yes.